protocols/virtualfund/helpers_test.go

```diff
@@ -0,0 +1,87 @@
+package virtualfund
```

Comment:
Changes:

This file is copy-past code from virtualfund-single-hop_test.go.

`actor` now has a `name` field, to help print friendly test messages.